docs/nightly/en/user-guide/overview.md

```diff
@@ -1,10 +1,66 @@
 # Overview
 
+Welcome to the user guide for GreptimeDB.
+This guide will help you explore each powerful feature of GreptimeDB.
+
+## SQL query example
+
+Let's start with a SQL query example.
+
+To monitor and analyze the performance and reliability of specific metrics over a recent period,
+engineers commonly use queries like the following:
```

Comment:
We can say user wants to join two data sources but nobody can use query like this because it was impossible to do this before.